### ISSUE TITLE
Potential issues with Metrolist listing form

### ISSUE THREAD

**Issue by duffy-james** [CityOfBoston/boston.gov-d8#2443]
The Metrolist team is reporting issues with the [Metrolist listing form](https://www.boston.gov/departments/neighborhood-development/metrolist/metrolist-listing-form). It looks like agents have been unable to successfully post on Metrolist.

The issue is specifically with the [listing form to salesforce](https://www.boston.gov/departments/neighborhood-development/metrolist/metrolist-listing-form), not the salesforce to Metrolist component, which looks to be working.

@padora-cob has done some recent work on updating our syncing with Salesforce, so hopefully this is all related. Adding this ticket to make sure it's accounted for.

**Comment by padora-cob** [comment]
I went through the whole Metrolist Listing system and code on my LOCAL last
night, but did not find any issues.

I then gave it a try on DEV and it also seemed to work, no error in any
of the logs.

Finally, I went on PROD and tried to submit a listing form. All worked as
expected. I was able to log-in to DND SalesForce Prod and the form and
information I had just submitted was there as expected.

I also noticed on DND SF Prod that multiple property managers had
successfully submitted new listings yesterday, on April 14th.

If the form is broken I will need more specifics at this point.
